src/lib/marshal/email-layout/writer.ts

```diff
@@ -0,0 +1,193 @@
+import path from "node:path";
```

Comment:
Can we add some basic tests for some of these writer module functions? Check out test/lib/marshal/workflow/writer.test.ts for examples.

Reply:
I did it in the latest commits